discord/ext/commands/pagination.py

```diff
@@ -0,0 +1,58 @@
+import discord, asyncio
+from typing import List, Union
+
+from discord import abc
+from discord.ui.button import button
+
+class Paginate(discord.ui.View):
+    """Creates a paginator for embeds that is navigated with buttons.
+    
+    Parameters
+    ------------
+    send_to: :class:`discord.abc.Messageable`
+    embed: List[:class:`discord.Embed`]
+    """
+
+    def __init__(self, send_to, embeds: List[discord.Embed]):
+        super().__init__()
+        self.send_to = send_to
+        self.embeds = embeds
+        self.current_page = 1
+
+    @discord.ui.button(label = "<", style = discord.ButtonStyle.green, disabled=True)
+    async def previous(self, button: discord.ui.Button, interaction = discord.Interaction):
+        page_count = len(self.embeds)
+
+        if not self.current_page:
+            self.current_page = 1
+        else:
+            self.current_page -= 1
```

Comment:
When would `self.current_page` be unset? Don't think the check is needed.

```suggestion
        self.current_page -= 1
```